lib/flipper/ui/views/feature.erb

```diff
@@ -273,14 +273,7 @@
               <form action="<%= script_name %>/features/<%= Flipper::UI::Util.escape @feature.key %>/expression" method="post" id="expression" class="row">
                 <%== csrf_input_tag %>
                 <input type="hidden" name="operation" value="enable">
-                <% if @feature.expression %>
```

Comment:
Are we good to remove
- lib/flipper/ui/views/_equal.erb
- lib/flipper/ui/views/_notequal.erb
- lib/flipper/ui/views/_greaterthan.erb
- lib/flipper/ui/views/_greaterthanorequalto.erb
- lib/flipper/ui/views/_lessthan.erb
- lib/flipper/ui/views/_lessthanorequalto.erb
- lib/flipper/ui/views/_comparable.erb

Wasn't exactly sure the plan for these compare operator partials